examples/single_column_radiation.jl

```diff
@@ -64,60 +86,97 @@ q₀ = 0.015    # surface specific humidity (kg/kg)
+ℐ_sw_gray = gray_radiation.downwelling_shortwave_flux
+ℐ_net_gray = ℐ_lw_up_gray + ℐ_lw_dn_gray + ℐ_sw_gray
+
+ℐ_lw_up_clear = clear_sky_radiation.upwelling_longwave_flux
+ℐ_lw_dn_clear = clear_sky_radiation.downwelling_longwave_flux
+ℐ_sw_clear = clear_sky_radiation.downwelling_shortwave_flux
+ℐ_net_clear = ℐ_lw_up_clear + ℐ_lw_dn_clear + ℐ_sw_clear
+
+ℐ_lw_up_2xco2 = high_co2_radiation.upwelling_longwave_flux
+ℐ_lw_dn_2xco2 = high_co2_radiation.downwelling_longwave_flux
+ℐ_sw_2xco2 = high_co2_radiation.downwelling_shortwave_flux
+ℐ_net_2xco2 = ℐ_lw_up_2xco2 + ℐ_lw_dn_2xco2 + ℐ_sw_2xco2
+
+set_theme!(fontsize=14, linewidth=2.5)
+
+# Format altitude ticks in km (but keep internal units in meters).
+z_ticks_km = 0:5:20
+z_ticks_m = ((z_ticks_km .* 1000), string.(z_ticks_km))
+
+fig = Figure(size=(1400, 800), fontsize=14)
+
+# Atmospheric state panels (top row)
+ax_T = Axis(fig[1, 1]; xlabel="Temperature (K)", ylabel="Altitude (km)",
+            yticks=z_ticks_m, xticks=200:25:300)
+ax_q = Axis(fig[1, 2]; xlabel="Specific humidity (kg/kg)", yticks=z_ticks_m)
+ax_H = Axis(fig[1, 3]; xlabel="Relative humidity (%)", yticks=z_ticks_m)
+
+# Radiation panels (bottom row) - one per component
+ax_lw_up = Axis(fig[2, 1]; xlabel="LW ↑ (W/m²)", ylabel="Altitude (km)", yticks=z_ticks_m)
+ax_lw_dn = Axis(fig[2, 2]; xlabel="LW ↓ (W/m²)", yticks=z_ticks_m)
+ax_sw_dn = Axis(fig[2, 3]; xlabel="SW ↓ (W/m²)", yticks=z_ticks_m)
+ax_net = Axis(fig[2, 4]; xlabel="Net flux (W/m²)", yticks=z_ticks_m)
+
+# Hide y-axis decorations on inner panels
+[hideydecorations!(ax, grid=false) for ax in (ax_q, ax_H, ax_lw_dn, ax_sw_dn, ax_net)]
+
+# Atmospheric state
+lines!(ax_T, T; color=:gray30)
+lines!(ax_q, qᵗ; color=:gray30)
+lines!(ax_H, 100ℋ; color=:gray30)
+
+# Colors for radiation schemes
+c_gray = :black
+c_clear = :dodgerblue
+c_2xco2 = :orangered
+
+# LW upwelling (positive)
+lines!(ax_lw_up, ℐ_lw_up_gray;  color=c_gray)
+lines!(ax_lw_up, ℐ_lw_up_clear; color=c_clear)
+lines!(ax_lw_up, ℐ_lw_up_2xco2; color=c_2xco2)
+
+# LW downwelling (negative, so we negate for display)
+lines!(ax_lw_dn, -ℐ_lw_dn_gray;  color=c_gray)
+lines!(ax_lw_dn, -ℐ_lw_dn_clear; color=c_clear)
+lines!(ax_lw_dn, -ℐ_lw_dn_2xco2; color=c_2xco2)
+
+# SW downwelling (negative, so we negate for display)
+lines!(ax_sw_dn, -ℐ_sw_gray;  color=c_gray)
+lines!(ax_sw_dn, -ℐ_sw_clear; color=c_clear)
+lines!(ax_sw_dn, -ℐ_sw_2xco2; color=c_2xco2)
+
+# Net flux
+lines!(ax_net, ℐ_net_gray;  color=c_gray)
+lines!(ax_net, ℐ_net_clear; color=c_clear)
+lines!(ax_net, ℐ_net_2xco2; color=c_2xco2)
+
+# Legend
+scheme_handles = [
+    LineElement(color=c_gray, linewidth=3),
+    LineElement(color=c_clear, linewidth=3),
+    LineElement(color=c_2xco2, linewidth=3),
+]
+scheme_labels = ["Gray", "Clear-sky (420 ppm)", "2×CO₂ (840 ppm)"]
+Legend(fig[0, :], scheme_handles, scheme_labels; orientation=:horizontal, framevisible=false, tellwidth=false)
 
 fig
+
+save("single_column_radiation.png", fig)
```

Comment:
Can this be removed?  I think in this position it'll also hide the display of the figure
```suggestion
```